src/Job/Job.php

```diff
@@ -84,27 +84,23 @@ public function setStateProperty($property, $value)
 
     public function jsonSerialize()
     {
-        return (object) ['timeLimit' => $this->timeLimit, 'result' => $this->getResult()];
+        return (object) [
+            'timeLimit' => $this->timeLimit,
+            'result' => $this->getResult()->jsonSerialize()
+        ];
     }
 
-    public static function hydrate($json)
-    {
+    /**
+     * Hydrate an object from the json created by jsonSerialize().
+     * You will want to override this method when implementing specific jobs.
+     * You can use this function for the initial JSON decoding by calling
+     * parent::hydrate() in your implementation.
+     *
+     * @param string $json
+     *   JSON string used to hydrate a new instance of the class.
+     */
+    public static function hydrate($json) {
```

Comment:
I am not sure how to handle this -- we want static functions on implementations of Job but the method itself doesn't do anything and can't really be tested. WHat I have here seems kind of dumb and a hack-y way to get to 100% test coverage, but not sure what the better alternative is.